.storybook/stories/Segments.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+import { Vector3 } from 'three'
+import { hilbert3D } from 'three-stdlib'
+import { withKnobs, number, color, boolean } from '@storybook/addon-knobs'
+
+import { Setup } from '../Setup'
+
+import { Segment, Segments, OrbitControls } from '../../src'
+import { useFrame } from '@react-three/fiber'
+
+export default {
+  title: 'Performance/Segments',
+  component: Segments,
+}
+
+const points = hilbert3D(new Vector3(0), 5).map((p) => [p.x, p.y, p.z]) as [number, number, number][]
+
+const colors = new Array(points.length).fill(0).map(() => [Math.random(), Math.random(), Math.random()]) as [
```

Comment:
You can write this as 

```
const colors = Array.from({ length: points.length }, () => [Math.random(), Math.random(), Math.random()])```

Reply:
That's great, however those two lines aren't used at all. I should remove them.